animations/staggered-animations/index.md

```diff
@@ -0,0 +1,428 @@
+  is responsive.
+-->
+<div class="embed-container"><iframe src="https://www.youtube.com/embed/0fFvnZemmh8?rel=0" frameborder="0" allowfullscreen></iframe></div>
+
+In the video, you see the following animation on a single widget,
+which begins as a bordered blue square with slightly rounded corners.
+The square:
+
+* Fades in
+* Widens
+* Becomes taller while moving upwards
+* Transforms into a bordered circle
+* Changes color to orange
+
+After running forwards, the animation runs in reverse.
+
+<aside class="alert alert-info" markdown="1">
+**New to Flutter?**
+You should first understand how to create a layout using Flutter’s
+widgets.  For more information, see [Building Layouts in
+Flutter](/tutorials/layout/).
+</aside>
+
+## Basic structure of a staggered animation
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What's the point?</b>
+
+* All of the animations are driven by the same
+  [AnimationController](https://docs.flutter.io/flutter/animation/AnimationController-class.html).
+* Regardless of how long the animations lasts in real time,
+  the controller's values vary between 0.0 and 1.0.
+* Break the animation down into
+  [Interval](https://docs.flutter.io/flutter/animation/Interval-class.html)s
+  specified within 0.0 and 1.0.
+  An Interval, for example, might run from 0.5 to 0.55.
+* Create a [Tween](https://docs.flutter.io/flutter/animation/Tween-class.html)
+  for each property that animates in an interval.
+  The Tween object specifies the start and end values for that property.
+  For example, a width tween might specify a starting width of 50 pixels
+  and an ending width of 150.
+* The tween produces an
+  [Animation](https://docs.flutter.io/flutter/animation/Animation-class.html)
+  object.
+</div>
+
+The app is essentially animating a Container whose decoration and size are
+animated. The Container is within another Container whose padding moves the
+inner container around and an Opacity widget that's used to fade everything
+in and out.
+
+The following diagram shows the Intervals used in the
+[basic_staggered_animation](https://github.com/flutter/website/tree/master/_includes/code/animation/basic_staggered_animation)
+example:
+
+<img src="images/StaggeredAnimationIntervals.png" alt="Diagram showing the interval specified for each motion." />
+
+Set up the animation as follows:
+
+* Create a Tween for each property being animated.
+* Call `animate` on the Tween, and pass in the Animation object.
+  This example uses
+  [CurvedAnimation](https://docs.flutter.io/flutter/animation/CurvedAnimation-class.html)
+  and specifies an eased curve.
+  See [Curves](https://docs.flutter.io/flutter/animation/Curves-class.html)
+  for other available pre-defined animation curves.
+* Specify the interval on the Animation's `curve` property.
+
+The following code shows how the `width` and
```

Comment:
The following code shows how the width and borderRadius animations are constructed.

They're used to configure the widget built by the AnimatedBuilder in StaggerAnimation.